### ISSUE TITLE
Maven dependencies should not be variables in readme?

### ISSUE THREAD

**Issue by Johnstedt** [paul-hammant/qdox#49]
I'm new to maven but this bit in the readme should have the latest id's and not a variable, right?

 	<dependency>
  	   <groupId>${project.groupId}</groupId>
  	   <artifactId>${project.artifactId}</artifactId>
  	   <version>${project.version}</version>
  	 </dependency>

**Comment by rfscholte** [comment]
Yes, this page should be generated, updating these values so you always see the most recent version. Must have gone wrong after migrating to github.

FYI, values should be:

    <dependency>
      <groupId>com.thoughtworks.qdox</groupId>
      <artifactId>qdox</artifactId>
      <version>2.0-M9</version>
    </dependency>

**Comment by Johnstedt** [comment]
Thank you very much!
